src/component/registry/types.ts

```diff
@@ -220,18 +228,25 @@ export type Font = StyleWithOpacity & {
  */
 export type Fill = StyleWithOpacity & {
   /**
-   * Possible values are all HTML color names or HEX codes, as well as special keywords such as ‘swimlane‘,
-   * ‘inherit’ to use the color code of a related cell.
+   * Possible values are all HTML color names or HEX codes, as well as special keywords such as:
+   * - `default` to use the color defined in the BPMN element default style
+   * - `none` for no color
    */
-  color?: string | 'swimlane' | 'inherit';
```

Comment:
**question:** 

For my information, why did some values, available on `mxgraph`, has been removed ?

Reply:
this is explained in the description of the PR